.gitattributes

```diff
@@ -0,0 +1,8 @@
+# Ensure that Git will always convert line endings to LF on checkout
+* text=auto
+# * text eol=lf # this affected pngs and such...
```

Comment:
Super minor, I'd just remove this comment! It makes sense to me because we talked, but probably just confuses others!
I might also change the comment on line 1 to something like
Allow git to convert line endings according to its default rules
Based on: [.gitattributes docs](https://www.git-scm.com/docs/gitattributes#Documentation/gitattributes.txt-Settostringvalueauto)

Reply:
Ah yeah, don't want comments to be confusing... Fixed